SPL/SPL.php

```diff
@@ -1752,7 +1752,7 @@ public function natcasesort(): bool {}
      * @param string $data <p>
      * The serialized <b>ArrayObject</b>.
      * </p>
-     * @return void The unserialized <b>ArrayObject</b>.
```

Comment:
Could you please clarify why you want to remove this comment? It exists at https://www.php.net/manual/en/arrayobject.unserialize.php

Reply:
a comment behind `return typeX` describes what the method returns. since this method doesn't return anything - aka `void` - the description which reads like a `unserialized ArrayObject` would be returned is wrong.

Reply:
Thank you. I've also reported this as https://github.com/php/doc-en/pull/1101

Reply:
> Thank you. I've also reported this as [php/doc-en#1101](https://github.com/php/doc-en/pull/1101)

good catch, thank you!